### PR TITLE
apply the same patch to DockerfileHub for docker image publishing

### PR DESCRIPTION
**Pull request by itaru2622**
followup #225

I noticed that #225 is missing in the latest image in dockerhub.

### CODE REVIEW

**Review by litong01 — CHANGES_REQUESTED** [review]
why are we removing ansible? that is the key component, without ansible, how will the playbook run?

File: DockerfileHub [hunk 1]
```diff
@@ -16,15 +16,17 @@ LABEL maintainer="litong01@us.ibm.com"
...
     if [ ! -e /usr/bin/python ]; then ln -sf python3 /usr/bin/python ; fi && \
-    mkdir -p /usr/lib/python3.8/site-packages/Crypto/Random/Fortuna
+    mkdir -p /usr/lib/python3.8/site-packages/Crypto/Random/Fortuna \
+             /usr/lib/python3.8/site-packages/ansible/plugins
 
 COPY --from=build /tmp/minifabric /home
 COPY --from=build /tmp/plugins /usr/lib/python3.8/site-packages/ansible/plugins
 COPY --from=build /tmp/pypatch /usr/lib/python3.8/site-packages/Crypto/Random/Fortuna
 RUN rm -rf /var/cache/apk/* && rm -rf /tmp/* && apk update && \
-    pip install openshift==0.11.2 requests google-auth && \
```
Comment (itaru2622) [inline DockerfileHub:27]: ansible is still here,  line 28.
patch is the same manner as Dockerfile (by #225)

**Review by itaru2622 — COMMENTED** [review]
ansible is still there in end of line 28
```
pip install requests google-auth openshift ansible==2.9.* && \
```
patch is the same manner as Dockerfile (by #225)

**Review by litong01 — CHANGES_REQUESTED** [review]
I do not see where it gets added back.

**Review by litong01 — APPROVED** [review]
I saw it now . Thx

**Pull request merged**